tools/static-users-generator/scripts/config_replace.sh

```diff
@@ -12,6 +12,19 @@ PLACEHOLDER="#__STATIC_PASSWORDS__"
 
 NEWLINE="%"
 
+# check if kubectl works as expected. Minikube stop and then minikube start cases temporary unavailability of the api-server)
+while :
+do
+  if [[ $(kubectl get secrets -l dex-user-config=true --all-namespaces -o json) ]]
+    then
+      echo "api-server available via kubectl"
+      break
+    else
+      echo "api-server not available via kubectl - waiting 5s..."
+      sleep 5
```

Comment:
Could you limit the number of retries, so the pod crashes in case of more serious errors?